### ISSUE TITLE
FortiusANT  window 10 error

### ISSUE THREAD

**Issue by Roguish000** [WouterJD/FortiusANT#343]
trying to start FortiusANT and get   'this app can't run on your PC'
runnning Windows 10 Pro, 20H2

download code 12-5-21, windows executable

sure would like to get it running.
trying on a Bushido.

thanks

**Comment by WouterJD** [comment]
Please provide some more info, not seen before

**Comment by Roguish000** [comment]
thanks for the quick reply.
What info?  I will do my best to get it all.....

**Comment by WouterJD** [comment]
Welcome to the FortiusANT community
- - -
I'm always curious to know who I communicate with, where FortiusANT is used and what configuration is used.
Please tell me what bundle did you buy, and what brake and what head unit do you use?
I would therefore appreciate that you introduce yourself; perhaps leave a comment under issue #14.
You could also consider to visit [the sponsor page](https://github.com/sponsors/WouterJD)
- - -

Could you please include a screen print of the console (black window) showing the exact messages.
Copy/paste the text would even be handier)

**Comment by WouterJD** [comment]
Google will send you [to](https://helpdeskgeek.com/windows-10/how-to-fix-this-app-cant-run-on-your-pc-in-windows-10/) and most likely the first option is what is hapening here.

I assume you run the windows executable...
It seems that you do not have sufficient rights to run the executable. The executable contains an embedded python runtime to execute the code of FortiusAnt and it seems this cannot be installed.
Either start the executable as administrator, give your account administrator rights or start as administrator

**Comment by Roguish000** [comment]
Thanks for the response...
I am in the San Francisco Bay Area.
I purchase the Bushido approximately 6 years ago,  Did not spare a dime, so got TTS 4 premium, and even a BlackTrack (the steering platform.)  And a new (at that time) Asus Q550L laptop, with dedicated graphics card.  (I also used the laptop for engineering design with Solidworks.)  TTS4 is up to the absolute latest rev.  Even paid for the Google Earth support.....

I have complete admin on the laptop, and do a fair bit of computer using, but not programming.
First option I tried was to 'run as administrator' as I've been down that hole many times.

I just went through the link you gave above for HelpDeskGeek.  The only thing I have not done is go into developer mode, otherwise I should have privileges. Checked all that for the .exe file.....

Not sure what other info I can provide.

Does it need to run in a compatibility mode?  Windows 7 perhaps?  Specific screen size and resolution?

posted the error message here.  hope it comes through.

Thanks again.  Hope we can get it all going........

![Image 6](https://user-images.githubusercontent.com/42012033/144767935-48174b29-114f-407c-8109-a0b90a8a99ff.png)
